web/src/components/merchant/listings/index.tsx

```diff
@@ -67,7 +67,10 @@ const ListingsPage: React.FC = () => {
               </AddListingButton>
             </ButtonContainer>
             <ListingsContainer>
-              <ListingCollection listings={listings} />
+              <ListingCollection
+                listings={listings}
+                listingRootPath="/merchant/"
```

Comment:
use constant

Reply:
@ifhuang Okay! Made this change in PR #196